environment.yml

```diff
@@ -9,6 +9,8 @@ dependencies:
     - nltk=3.2.5
     - scikit-learn=0.19.1
     - pandas=0.23.0
+    - bokeh=1.2.0
```

Comment:
Mind adding comments here on where this is used? I want to make it easier for users to skip/ignore/delete items here in case conflicts appear later. (I'm doing the same in the XLNet PR.)

Reply:
Done. Added comments for some older ones too.

Though, since this is a fully-specified environment, there really shouldn't be any package conflicts - that's on us to fix.